xla/service/collective_quantizer.cc

```diff
@@ -23,13 +24,15 @@ namespace {
 
 namespace m = match;
 
-// Describes the ops of a quantization, dequantization or plain type conversion
-// subgraph.
+// Holds the ops of a subgraph describing quantization (conversion to a narrower
+// type after scaling by a broadcasted scalar and clamping), dequantization
+// (scaling by a broadcasted scalar after type conversion to a wider type) or
+// plain type conversion.
 struct ConversionSubgraph {
-  HloInstruction* convert;
+  HloInstruction* convert = nullptr;
   HloInstruction* binary = nullptr;
-  HloInstruction* clamp;
-  HloInstruction* scale_bcast;
+  HloInstruction* clamp = nullptr;
+  HloInstruction* scale_bcast = nullptr;
   std::vector<HloInstruction*> unaries;
```

Comment:
Document what unaries are, e.g. "List of instructions following a dequantization or preceding a dequantization, in top-down order (operand-to-user)"